controllers/rcj_soccer_referee_supervisor/referee/supervisor.py

```diff
@@ -146,6 +146,17 @@ def draw_scores(self, blue: int, yellow: int):
             "Tahoma",  # Font
         )
 
+        self.setLabel(
```

Comment:
Do we need to be rewriting the team names in each tick? Cannot it happen once?

Reply:
It probably does not need to but this felt safer. But I can try to extract it out if you find that to be a show stopper.